src/vm-repair/azext_vm_repair/_validators.py

```diff
@@ -72,6 +72,10 @@ def validate_create(cmd, namespace):
     else:
         logger.debug('The source VM\'s OS disk is not encrypted')
 
+    if namespace.encrypt_recovery_key:
+        if not namespace.unlock_encrypted_vm:
+            raise CLIError('Recovery password is provided in the argument, but unlock_encypted_vm is not passed. Rerun command adding --unlock_encypted_vm.')
```

Comment:
```suggestion
            raise RequiredArgumentMissingError('Recovery password is provided in the argument, but unlock_encypted_vm is not passed. Rerun command adding --unlock_encypted_vm.')
```
Please use a more specific error type `RequiredArgumentMissingError` instead of `CLIError`